### PR TITLE
PostAffiliatePro Integration

### PR DESCRIPTION
**Pull request by ItsSudip**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/378)

### CODE REVIEW

File: integrations/integration_cname.js [hunk 1]
```diff
@@ -124,6 +124,10 @@ const commonNames = {
   quantumMetric: "QUANTUMMETRIC",
   quantummetric: "QUANTUMMETRIC",
   Quantum_Metric: "QUANTUMMETRIC",
+  PostAffiliatePro: "POSTAFFILIATEPRO",
+  Post_ffiliate_pro: "POSTAFFILIATEPRO",
```
[the remark this answers is not in the document]
Comment (ItsSudip) [reply integrations/integration_cname.js]: Resolved

File: integrations/integration_cname.js [hunk 2]
```diff
@@ -124,6 +124,10 @@ const commonNames = {
...
+  POST_AFFILIATE_PRO: "POSTAFFILIATEPRO",
+  postaffiliatepro: "POSTAFFILIATEPRO",
```
[the remark this answers is not in the document]
Comment (ItsSudip) [reply integrations/integration_cname.js]: Resolved

File: integrations/index.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ import * as Sentry from "./Sentry";
 import * as SnapPixel from "./SnapPixel";
 import * as TVSquared from "./TVSquared";
 import * as VWO from "./VWO";
-
```
[the remark this answers is not in the document]
Comment (ItsSudip) [reply integrations/index.js:43]: Resolved.

File: integrations/PostAffiliatePro/util.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const ScriptLoader = (url) => {
```
[the remark this answers is not in the document]
Comment (ItsSudip) [reply integrations/PostAffiliatePro/util.js]: I have used the ScriptLoader in /integrations. No changes are needed, the existing ScriptLoader is sufficient.

File: integrations/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (saikumarrs) [reply integrations/index.js:43]: Add a new empty line after Ln 42 and Ln 45

[the remark this answers is not in the document]
Comment (ItsSudip) [reply integrations/index.js:43]: Resolved.

File: integrations/PostAffiliatePro/browser.js [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    this.clickEvents = config.clickEvents;
+  }
+
+  init() {
+    logger.debug("===in init Post Affiliate Pro===");
+    if (!this.url) {
+      logger.debug("URL is missing");
+      return;
+    }
+    ScriptLoader("pap_x2s6df8d", this.url);
```
[the remark this answers is not in the document]
Comment (ItsSudip) [reply integrations/PostAffiliatePro/browser.js:33]: This is same for every customer.

**Review by yashasvibajpai — COMMENTED** [review]
LGTM.

**Pull request merged**